frontend/templates/views/2021/partials/extras/about/components-showcase.j2

```diff
@@ -1,33 +1,53 @@
+    <button [class]="accordionVisibility ? 'ap-button' : 'ap-button --inactive'" on="tap:AMP.setState({carouselVisibility: false, animationVisibility: false, accordionVisibility: true, autocompleteVisibility: false})" class="ap-button --inactive">{{ _('Accordion') }}</button>
+    <button [class]="autocompleteVisibility ? 'ap-button' : 'ap-button --inactive'" on="tap:AMP.setState({carouselVisibility: false, animationVisibility: false, accordionVisibility: false, autocompleteVisibility: true})" class="ap-button --inactive">{{ _('Autocomplete') }}</button>
   </div>
 
-  <div class="ap-components-showcase-demos">
-    <div class="ap-components-showcase-demo --carousel">
-      <div class="ap-components-showcase-demo-code">
+  <div class="ap-components-showcase-demos">  
+    {%for component in components %}
+    {%set visibilityName = component ~ "Visibility" %}
+    {%set componentLink = "/shared/components-showcase/amp-" ~ component ~ ".html" %}
+    {%set componentBind = visibilityName ~ " ? 'ap-components-showcase-demo --"~component ~ " show' : 'hide' " %}
+    {%set componentDoc = g.doc('/content/amp-dev/documentation/components/amp-'~component, locale=doc.locale).url.path %}
+    {%if component == components[0] %}
+      {%set componentClass = "ap-components-showcase-demo --" ~component~ " show" %}
+    {% else %}
+    {%set componentClass = "hide" %}
+    {%endif%}
+
+    <div [class]="{{componentBind | safe }}" class="{{componentClass | safe}}">
+      <div class="ap-components-showcase-demo-code" title={{"amp-"~component}}>
         <div class="ap-code-snippet">
-          {% set code_snippet = g.doc('/shared/components-showcase/amp-carousel.html').content %}
+          {% set code_snippet = g.doc(componentLink).content %}
           {{ doc.pod.markdown.convert('```html\n' + code_snippet + '\n```')|safe }}
         </div>
       </div>
 
-      <div class="ap-components-showcase-demo-ui">
-        <!-- Styled demo here -->
-      </div>
-    </div>
-    <div class="ap-components-showcase --accordion">
-      <div class="ap-components-showcase-demo-code">
-        <div class="ap-code-snippet">
-          {% set code_snippet = g.doc('/shared/components-showcase/amp-accordion.html').content %}
-          {{ doc.pod.markdown.convert('```html\n' + code_snippet + '\n```')|safe }}
+      <div class="ap-components-showcase-demo-ui {{component | safe}}" title={{"<amp-"~component~">"}}>
+
+      {%if component == 'animation' %}
+        <div class="rotate-image-window">
+          <amp-img id="animatedImage" width="450" height="300" layout="responsive" src="/static/samples/img/elephant.jpg" alt="Picture of an elephant"></amp-img>
+          <button on="tap:scaleUpImage.start" class="ap-button">Click to scale up the image</button>
```

Comment:
Also the example needs to be as concise as possible. This is just my opinion but I think naming the animation `scaleUpAnimation` would make things even more obvious. But @sebastianbenz definitely also has a opinion on those 🙂